pkgs/by-name/ni/nixos-container/package.nix

```
@@ -33,9 +33,14 @@ replaceVarsWith {
   };
 
   postInstall = ''
-    t=$out/share/bash-completion/completions
-    mkdir -p $t
-    cp ${./nixos-container-completion.sh} $t/nixos-container
+    bashcomp=$out/share/bash-completion/completions
+    fishcomp=$out/share/fish/vendor_completions.d
+
+    mkdir -p $bashcomp $fishcomp
+
+    cp ${./nixos-container-completion.sh} $bashcomp/nixos-container
+    cp ${./nixos-container-completion.fish} $fishcomp/nixos-container.fish
```

```suggestion
    installShellCompletion --cmd nixos-container \
      --bash ${./nixos-container-completion.sh} \
      --fish ${./nixos-container-completion.fish}
```
Also need `nativeBuildInputs = [ installShellFiles ];`

---

huh, didnt now that existed. thanks!